zuul.d/end-to-end.yaml

```diff
@@ -32,6 +32,7 @@
     vars:
       cifmw_extras:
         - '@scenarios/centos-9/ci-build.yml'
+        - '@scenarios/centos-9/ceph_backends.yml'
```

Comment:
Note that with this change, cifmw-end-to-end will configure cinder, glance and manila backends, and will run more tempest tests.